content/docker-id/_index.md

```diff
@@ -34,9 +36,25 @@ Your Docker ID becomes your username for hosted Docker services, and [Docker for
 >
 > You have limited actions available until you verify your email address.
 
+### Sign up with Google or GitHub
+
+1. Go to the [Docker sign-up page](https://hub.docker.com/signup/).
+
+2. Select your social provider, Google or GitHub.
+
+3. Select the social account you want to link to your Docker account.
+
+4. Select **Authorize Docker** to allow Docker to access your social account information and be re-routed to the sign-up page.
+
+5. Enter a username.
+
+    Your Docker ID must be between 4 and 30 characters long, and can only contain numbers and lowercase letters. Once you create your Docker ID you can't reuse it in the future if you deactivate this account.
+
+6. Select **Sign up**.
+
 ## Sign in
 
-Once you register and verify your Docker ID email address, you can sign in to [Docker Hub](https://hub.docker.com).
+Once you register and verify your Docker ID email address, you can sign in to [your Docker account](https://login.docker.com/u/login/). You can sign in with your email address or username and password, or with your Google or GitHub account. See [Sign in with your social provider](#sign-in-with-your-social-provider).
```

Comment:
`You can sign in with your email address or username and password, or with your Google or GitHub account.`

It makes sense, but I had to read it a couple times because of the multiple conjunctions.
I have no suggestion on how to change it and keep the same meaning. This may be more simple, but might also change your intended meaning, `You can sign in with your email address, username, or social provider.`

Reply:
Yeah I wasn't totally sure about the wording for this either. Would this work:
```suggestion
Once you register and verify your Docker ID email address, you can sign in to [your Docker account](https://login.docker.com/u/login/). You can sign in with your email address (or username) and password. Or, you can sign in with your social provider. See [Sign in with your social provider](#sign-in-with-your-social-provider).
```